EMalgo_continous_categorical_data.R

```diff
@@ -0,0 +1,124 @@
+#Description ####
+#EM Algorithm for continous variables and ctegorical variables
+#Model Based Learning project
+#Groupe : Florent Jakubowski et Marc Telly 
+#Date : 2020-11-15
+#For mor informations read the README
+#All functions should be tested with unit tests using testthat package
+#To understand our way of writting code see the book CleanCode. 
+
+# packages ####
+library(bayess)
+library(mvtnorm)
+
+
+# FLO function TransformCategoricalVectorToBinaryMatrice ####
+TransformCategoricalVectorToBinaryMatrice <- function (categoricalVector) {
+  vectorUniqueValues <- unique(categoricalVector)
+  binaryMatrice <- matrix(0, nrow=length(categoricalVector), ncol=length(vectorUniqueValues))
+  for (i in 1:length(categoricalVector)) {
+    binaryMatrice[i, which(vectorUniqueValues == categoricalVector[i])] <- 1
+  }
```

Comment:
Maybe we can do this with a lapply ?

Reply:
Dejà avant d'optimiser vec lapply il faudrait que cette function fasse tout ce dont on veut d'elle

Reply:
c'est à dire ? tu peux être plus explicite @MARCTELLY